Added 12 new Q&A pairs to the Support knowledgebase section (#1081)
wandb/docs: content/support/how_can_i_delete_my_user_account.md

Reviewer: @luisbergua Should we add a clear warning that once deleted the account is unrecoverable?

Reviewer 2: And maybe how long it takes to take effect?

Author: Added!

---
title: How can I delete my user account?
toc_hide: true
type: docs
tags:
  - user management
---

> You can delete your user account from the bottom of your `<base-url>/settings` page.

Reviewer 2: ```suggestion
Delete your user account by clicking **Delete account** in your [user settings]({{< relref "/guides/models/app/settings-page/user-settings.md#delete-your-account" >}}).
```
